### ISSUE TITLE
too many /ping logs

### ISSUE THREAD

**Issue by kernel164** [pomerium/pomerium#776]
## What happened?

too many /ping logs in pomerium-authenticate & pomerium-proxy process

## What did you expect to happen?

/ping should not be logged as info atleast.

## What's your environment like?

- Pomerium version (retrieve with `pomerium --version` or `/ping` endpoint):

{"level":"info","version":"0.8.0-1589338260+80166bc","time":"2020-05-26T09:17:20Z","message":"cmd/pomerium"}

using pomerium-helm version 9.1.0

- Server Operating System/Architecture/Cloud:

AWS EKS 1.16

## What did you see in the logs?

```logs
{"level":"debug","ip":"10.180.57.159","user_agent":"kube-probe/1.16+","req_id":"cbb01711-f480-5687-a096-ab104e463a0b","duration":0.01858,"size":61,"status":200,"method":"GET","service":"authenticate","host":"10.180.50.243:443","path":"/ping","time":"2020-05-26T11:29:16Z","message":"http-request"}
```

**Comment by kernel164** [comment]
Oops sorry, I got confused with nginx-ingress-controller kube-probe ping logs. It is indeed debug level in pomerium.

**Comment by kernel164** [comment]
Reopening this ticket. I do see ping logs in info level for proxy and authenticate service

```
pomerium-authenticate-86df9d6856-6cnvq pomerium 2020-06-03T01:50:56.087864598Z {"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"10.180.58.243:443","path":"/ping","user-agent":"kube-probe/1.16+","referer":"","forwarded-for":"","request-id":"90a4eb5c-9598-4c1d-86ea-55506075f8b3","duration":0.499057,"size":61,"response-code":200,"response-code-details":"via_upstream","time":"2020-06-03T01:50:56Z","message":"http-request"}
pomerium-proxy-89f67cb5d-bvcvj pomerium 2020-06-03T01:50:56.353460091Z {"level":"info","service":"envoy","upstream-cluster":"pomerium-control-plane-http","method":"GET","authority":"10.180.57.135:443","path":"/ping","user-agent":"kube-probe/1.16+","referer":"","forwarded-for":"","request-id":"26dd100c-916c-4932-a8e8-80f9b7c79273","duration":0.410677,"size":61,"response-code":200,"response-code-details":"via_upstream","time":"2020-06-03T01:50:56Z","message":"http-request"}
```

**Comment by rspier** [comment]
the log level is fixed:

internal/controlplane/grpc_accesslog.go line 24:
			evt := log.Info().Str("service", "envoy")

It would be possible, but more than a little hacky to downgrade that to Debug() 

```
var evt *zerolog.Event
if entry.GetRequest().GetPath() == "/path" {
  evt = log.Debug()
} else {
  evt = log.Info()
}
```

**Comment by desimone** [comment]
\cc @rspier @kernel164 this should be fixed on master with #1368

**Comment by rspier** [comment]
Thanks!
